net-libs/libcloudproviders/libcloudproviders-0.3.1.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+VALA_USE_DEPEND=vapigen
+
+inherit gnome.org meson vala
+
+DESCRIPTION="DBus API that allows cloud storage sync clients to expose their services"
+HOMEPAGE="https://gitlab.gnome.org/World/libcloudproviders"
+
+LICENSE="LGPL-3"
+SLOT="0"
+IUSE="gtk-doc +introspection vala"
+
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~riscv ~sparc ~x86 ~amd64-linux ~x86-linux"
```

Comment:
We have to ask architecture teams to test on platforms we can't test on. Change this to just `KEYWORDS="~amd64"`.